tests/version_check.py

```diff
@@ -24,21 +24,20 @@
 
 # handle possible issues & give proper return codes
 if b'Python 3.8 or higher' in stdout or b'successfully created' in stdout:
+    status = {True: "failed", False: "succeded"}
     if actual_fail == expect_fail:
-        print("Python {} {} as expected".format(
-            platform.python_version(),
-            "failed" if expect_fail else "succeeded"))
+        print(f"Python {platform.python_version()} {status[expect_fail]} as expected")
     else:
-        print("Python {} should have {}, but actually {}".format(
-            platform.python_version(),
-            "failed" if expect_fail else "succeeded",
-            "failed" if actual_fail else "succeeded"))
+        print(
+            f"{platform.python_version()} should have {status[expect_fail]},",
+            f"but actually {status[actual_fail]}",
```

Comment:
please add "Python" before printing the version

Reply:
I missed that. fixed